README.rst

```diff
@@ -65,6 +65,13 @@ defining defaults or reading from multiple files:
     # if set, load_name will take a look at environment variables like
     # APP_FOO_BAR and APP_FOO_BAZ, mixing those in as foo.bar and foo.baz
 
+    # the default load order can be overridden if necessary:
+
+    configuration = confidence.load_name('app', load_order=confidence.loaders(
```

Comment:
We could consider adding some load orders that are in demand with an alias (the "homedir takes precedence" load order for example).

Reply:
Hmm, true, but considering the amount of options there are for this (ignore system-wide settings, ordering of user / environment / …), not sure these would offer much aside from confusion of a call to `loaders()`.

Reply:
In any case, these can always be added later; load order override feature can be merged as-is.